src/System.Windows.Forms/src/System/Windows/Forms/MonthCalendar.MonthCalendarAccessibleObject.cs

```diff
@@ -503,6 +503,16 @@ private bool GetCalendarGridInfo(ref MCGRIDINFO gridInfo)
                 return User32.SendMessageW(_owner, (User32.WM)MCM.GETCALENDARGRIDINFO, IntPtr.Zero, ref gridInfo) != IntPtr.Zero;
             }
 
+            public int GetCalendarCount()
+            {
+                // Do not use this if gridInfo.dwFlags contains MCGIF_NAME;
+                // use GetCalendarGridInfoText() instead.
+
+                int calendarCount = (int) User32.SendMessageW(_owner, (User32.WM)MCM.GETCALENDARCOUNT, IntPtr.Zero, IntPtr.Zero);
```

Comment:
```suggestion
                int calendarCount = (int)User32.SendMessageW(_owner, (User32.WM)MCM.GETCALENDARCOUNT);
```

Please format the code to be consistent with the existing code.